static/app/views/performance/newTraceDetails/traceDrawer/traceProfilingLink.ts

```diff
@@ -26,7 +26,7 @@ function toDate(value: unknown): Date | null {
  */
 export function makeTraceContinuousProfilingLink(
   value: TraceTreeNode<TraceTree.NodeValue>,
-  transaction: EventTransaction,
```

Comment:
Modified this so we dont tie it to txn even anymore (it is now also available on the spans)